client/src/components/mainPage/map/map.component.ts

```diff
@@ -24,13 +20,11 @@ export default class MapComponent extends Vue {
   el: any = "";
   selectedMarker: any = null;
 
-
   mapMarker = require("../../../assets/mainPage/default-marker.png");
   clickMapMarker = require("../../../assets/mainPage/click-marker.png");
```

Comment:
```suggestion
  clickedMapMarker = require("../../../assets/mainPage/click-marker.png");
```
- 수동태로 변수명 고쳐도 좋을 것 같네요!
- 그리구 변수명을 카카오 api에서 그대로 쓰는 부분들이 많은데 주석이 필요하지 않을 수 있게 조금씩 변경하면 좋을 것 같습니다!